elastic/logs/challenges/logging-streams.json

```diff
@@ -0,0 +1,331 @@
+                "params": {
+                  "field": "logs"
+                }
+              }
+            },
+            {
+              "pipeline": {
+                "name": "logs@stream.reroutes",
+                "ignore_missing_pipeline": true
+              }
+            }
+          ]
+        }
+      }
+    },
+    {
+      "name": "logs-index-template",
+      "operation": {
+        "operation-type": "create-composable-template",
+        "template": "logs@stream",
+        "body": {
+          "index_patterns": [
+            "logs"
+          ],
+          "template": {
+            "settings": {
+              "index": {
+                "default_pipeline": "logs@stream.processing"
+              }
+            },
+            "mappings": {
+              "properties": {
+                "stream.name": {
+                  "type": "constant_keyword",
+                  "value": "logs"
+                }
+              }
+            }
+          },
+          "composed_of": [
+            "logs@stream.layer"
+          ],
+          "priority": 1000,
+          "version": 1,
+          "_meta": {
+            "managed": true,
+            "description": "The index template for logs stream"
+          },
+          "data_stream": {
+            "hidden": false,
+            "allow_custom_routing": false
+          }
+        }
+      }
+    },
+     {
+      "name": "setup",
+      "operation": {
+        "operation-type": "put-pipeline",
+        "id": "logs@custom",
+        "body": {
+          "description": "Sends all data to the 'logs' index. This is easier than changing the index name in the source.",
+          "processors": [
+            {
+              "reroute": {
+                "destination": "logs"
+              }
+            }
+          ]
+        }
+      }
+    },
+    {
+      "name": "bulk-index",
+      "operation": {
+        "operation-type": "raw-bulk",
+        "param-source": "processed-source",
+        "time-format": "milliseconds",
+        "profile": "fixed_interval",
+        "bulk-size": {{ p_bulk_size }},
+        "detailed-results": true
+      },
+      "clients": {{ p_bulk_indexing_clients }},
+      "ignore-response-error-level": "{{error_level | default('non-fatal')}}"
+      {% if p_throttle_indexing %},
+      "schedule": "timestamp-throttler",
+      "max-delay-secs": 1
+      {% endif %}
+    },
+      {
+      "name": "compression-stats",
+      "operation": {
+        "operation-type": "compression-statistics",
+        "data-stream": "logs"
+      }
+    },
+    {
+      "name": "force-merge-datastreams",
+      "operation": {
+        "operation-type": "force-merge",
+        "index": "logs",
+        "mode": "polling",
+        "max-num-segments": 1,
+        "request-timeout": 60
+      }
+    },
+    {
+      "name": "compression-stats-after-merge",
+      "operation": {
+        "operation-type": "compression-statistics",
+        "data-stream": "logs"
+      }
+    },
```

Comment:
Do you care about compression stats and force merging? If not, please simplify removing those. The data streams and index pattern are incorrect here, see other comments.